pom.xml

```diff
@@ -30,6 +30,7 @@
         <surefire.system.args>-Xms512m -Xmx512m ${modular.jdk.args}</surefire.system.args>
         <!-- Plugins versions -->
         <version.org.jacoco.plugin>0.7.9</version.org.jacoco.plugin>
+        <version.org.wildfly.jar.plugin>2.0.0.Beta1</version.org.wildfly.jar.plugin>

```

Comment:
2.0.0.Beta4 was released yesterday.

Reply:
Thanks, I will update it and start new CI run

Reply:
Updated to 2.0.0.Beta5 as we discussed on google chat ...

Reply:
Marking this thread as resolved, let me know if you have some objections ...